paymentsheet/src/main/java/com/stripe/android/paymentsheet/AddPaymentMethodsAdapter.kt

```diff
@@ -27,36 +43,21 @@ internal class AddPaymentMethodsAdapter(
     override fun getItemCount() = paymentMethods.size
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): AddPaymentMethodViewHolder {
-        return AddPaymentMethodViewHolder(parent)
-            .apply {
-                val targetWidth = parent.measuredWidth - parent.paddingStart - parent.paddingEnd
```

Comment:
I really like this change to do the calculation in a separate function.  Really helps readability!